app/src/main/res/menu/multimedia_popup.xml

```diff
@@ -0,0 +1,17 @@
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.menu.multimedia_popup` appears to be unused
 Unused resources make applications larger and slow down builds.

Reply:
This will be of use in my next pull request.